src/pages/myNotes/myNotes.jsx

```diff
@@ -0,0 +1,32 @@
+import { Box } from '@material-ui/core';
+import React, { useState } from 'react';
+import { NOTES_LIST } from '../../assets/data/index';
+import Note from './note';
+import NoteText from './noteText';
+import useStyles from './styled';
+
+const MyNotes = () => {
+  const text = 'Select note to display';
+  const [defaultText, activeText] = useState({
+    title: text,
+    description: '',
+    date: '',
+  });
+  const changeText = (title, description, date) => {
+    activeText({ title, description, date });
+  };
+  const classes = useStyles();
+  const [notActive, active] = useState(0);
```

Comment:
`active` it's not a function, but `setActive` it is. And there is no need to define an addition function `changeState`, you can call just `setActive(id)` from `Note`